lib/src/enums/api_version.dart

```diff
@@ -0,0 +1,7 @@
+enum ApiVersion {
+  version2019(value: "2019-05-29");
```

Comment:
I think it's better to use `version` instead of `version2019` so that when we need to bump the version, we don't have add a new enum here.

Reply:
Maybe just make it hard coded without an enum, as an enum is usually used to provide multiple options, but if we are not then i will make it hard coded in the http client itself.